recipe/meta.yaml

```diff
@@ -8,13 +8,21 @@ source:
     fn: flann-{{ version }}.tar.gz
     url: https://github.com/mariusmuja/flann/archive/{{ version }}.tar.gz
     sha256: 59a9925dac0705b281496ae52b5dfd79d6b69316d37015e3d3b38c859bac4f2f
+    
+    patches:
+      # Borrowed from https://github.com/mariusmuja/flann/issues/82
+      - win64_serialization.patch  # [win64]
```

Comment:
Oops. 😳

Fortunately it looks like a simple patch. We should be able to generate a new one based off of it (unless the release has made this unnecessary).